regulations/views/mixins.py

```diff
@@ -19,7 +19,7 @@ def get_context_data(self, **kwargs):
             sidebars.append(
                 self.build_sidebar_context(
                     class_or_class_path,
-                    context['label_id'],
+                    f"{context['part']}-{context['section']}",
```

Comment:
There should be a way to construct a mixin that will populate a `citation` field with what is constructed here.

Reply:
https://jiraent.cms.gov/browse/EREGCSC-279